From 834472a72adfc922bf47e34cc6ff155956f9269c Mon Sep 17 00:00:00 2001
From: anonym <anonym@riseup.net>
Date: Sat, 19 Nov 2016 11:59:21 +0100
Subject: [PATCH] AppArmor profile: allow gtk-engines-bixbuf and themes.

This will give Icedove a native look in e.g. KDE (if something like
gtk3-engines-breeze is installed), instead of a look reminding us of the
aestethics found in Windows 95.
---
 debian/apparmor/usr.bin.icedove | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/debian/apparmor/usr.bin.icedove b/debian/apparmor/usr.bin.icedove
index ba023cd..96abeec 100644
--- a/debian/apparmor/usr.bin.icedove
+++ b/debian/apparmor/usr.bin.icedove
@@ -175,6 +175,10 @@ profile icedove /usr/lib/icedove/icedove {
   /bin/uname Uxr,
   /usr/bin/locale Uxr,
 
+  # Theme support for desktop environments not based on GTK
+  /usr/lib/@{multiarch}/gtk-*/*/engines/libpixmap.so* mr,
+  /usr/share/themes/** r,
+
   /usr/bin/gpg Cx -> gpg,
 
   profile gpg {
-- 
2.10.2

